styles/styles.css

```diff
@@ -50,20 +50,35 @@ div[class*='-up'] .con-block.has-bg.text-block.text-list,
   margin-bottom: var(--spacing-xxs);
 }
 
+ /* CaaS Height Styles  */
+ .caas-spacing-m {
+  min-height: 500px;
+}
+
+.milo-iframe.webinars-fix {
+  min-height: 4463px;
```

Comment:
These values will always be prone to error and they do leave a lot of space between page content and footer, such as
<img width="408" alt="Space before footer" src="https://github.com/adobecom/bacom/assets/11267498/7bdd75bd-be63-4aac-bb1e-a9ac94bc3057">

Is there no way to add some logic to the frame itself and the page? Using `window.frameElement` to see if the content is within an iframe? Using `postMessage` - https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage ? Plain ol' `parent` object? Then you could get the height of the content from the iframe and adapt its height attributes to match.

Reply:
I 100% agree this is not a good solution, and it is meant to fix only 1 page https://business.adobe.com/resources/webinars.html. 

My initial suggestions to our business partners were to a) ask us to build the components in Milo they need to author this page and 2) if the content needs to come from the iframe source (which is a third party), then redirect to that page. 

I'll se what I can do from a JS standpoint.

Reply:
Closing my eyes while I approve, then 🙈  Hopefully, if more such pages pop up, a better long-term/maintainable solution can be found.

Reply:
Thank you. I appreciate you bringing this up though. I will mention this at our standup. It is good to highlight that the "fix it for now" solution that got requested here is antithetical to the way in which we work on Milo and other projects.

For what it is worth, I did look into `window.frameElement`, `postMessage`, and `parent`, but because the iframe code comes from a third party to which we don't have access (i'll triple check on that), we can't use them effectively. The iframe does send a postMessage, but not with the height. The iframe loads the whole page so slowly all our code executes before it loads.